src/include/optimizer/stats/value_condition.h

```diff
@@ -0,0 +1,50 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// value_condition.h
+//
+// Identification: src/include/optimizer/stats/value_condition.h
+//
+// Copyright (c) 2015-16, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "common/logger.h"
+#include "type/value.h"
+#include "type/types.h"
+
+namespace peloton{
+namespace optimizer{
+
+//===--------------------------------------------------------------------===//
+// ValueCondition
+// SELECT * FROM table WHERE [id = 1] <- ValueCondition
+//===--------------------------------------------------------------------===//
+class ValueCondition {
+public:
+  oid_t column_id;
+  std::string column_name;
+  ExpressionType type;
+  type::Value value;
+
+  // Only with id.
+  ValueCondition(oid_t column_id, ExpressionType type, const type::Value& value)
```

Comment:
why not define column_name="" as a default parameter to combine these 2 functions into 1. So if you recieve a column name you use that else it is "".